src/system/time/services/ntp.cpp

```diff
@@ -14,7 +14,10 @@ bool TimeProviderNTP::init()
 {
     logger << LOG_INFO << "Initializing NTP..." << EndLine;
     timeClient.begin();
-    timeClient.update();
+    if(this->UpdateNtp())
+    {
+        return false;
+    }
     return true;
```

Comment:
You can just return `this->updateNTP()`

Reply:
Changes implemented were enough to solve the problem?

Reply:
I was reviewing and realized that the problem still there:
Readed time: 230-73-137T237:210:215Z